src/style.css

```
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+header {
```

use class selectors for styling instead
```suggestion
.header {
```

---

Sure, it has been added in the commit 497a8361cee2ecad9025122a261d5a3ac16db94d